spec/factories/authentication_tokens.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :authentication_token do
+    hashed_token { "MyString" }
```

Comment:
Can we use

```
Digest::SHA256.hexdigest(Time.zone.now)
```

or something what will give us something a bit more real/some uniqueness